parity-util-mem/README.md

```diff
@@ -4,7 +4,6 @@ Collection of memory related utilities.
 
 ## Features
```

Comment:
Can you add a recommended way of using this crate to, for example, replace code that uses `heapsizeof` today and wishes to stop? :)